package.json

```diff
@@ -15,6 +15,7 @@
     "babel-plugin-react-css-modules": "^3.4.2",
     "chartist": "^0.11.0",
     "ckeditor": "github:indico/indico-ckeditor#build-4.7.3",
+    "click": "^0.1.0",
```

Comment:
this doesn't look intentional, at least your code isn't using it anywhere. Also, the package is 5 years old